crates/turborepo-ui/src/tui/table.rs

```diff
@@ -53,7 +53,7 @@ impl TaskTable {
             .unwrap_or_default()
             // Task column width should be large enough to fit "Task" title
             // and truncate tasks with more than 40 chars.
-            .clamp(4, 40) as u16;
+            .clamp(14, 40) as u16;
```

Comment:
Was this bump intentional?

Reply:
The idea was that the minimum width needs to be the width of the "up/down to navigate" line now instead of the "Task" line. Wrong spot?

Reply:
Right spot, I just think it needs to happen a few other places as well.

Reply:
Only thing that stood out to me was this constraint, bumped it: https://github.com/vercel/turbo/pull/8499/files#diff-17d339c7c94986602475a5018ddff2b58833d1227bd33434eba1214976c4fddfL258

Anywhere else?